src/schemas/raster.schema.json

```diff
@@ -4,22 +4,71 @@
   "title": "Vitessce image data",
   "type": "object",
   "definitions": {
-    "raster": {
+    "imageTransform": {
       "type": "object",
-      "additionalProperties": true,
-      "required": ["sample", "tileSource"],
+      "additionalProperties": false,
+      "required": ["scale", "translate"],
       "properties": {
-        "sample": { "type": "number" },
-        "tileSource": {
-          "oneOf": [
-            { "type": "object" },
-            { "type": "string" }
-          ]
+        "scale": { "type": "integer" },
+        "translate": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["y", "x"],
+          "properties": {
+            "y": { "type": "integer" },
+            "x": { "type": "integer" }
+          }
         }
       }
+    },
+    "imageDimensions": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["field", "type", "values"],
+        "properties": {
+          "field": { "type": "string" },
+          "type": {
+            "type": "string",
+            "examples": ["quantitative", "nominal", "ordinal", "temporal"]
+          },
+          "values": {
+            "type": ["array", "null"],
```

Comment:
Less special case checking down stream if this is always an array, perhaps of zero length?

Reply:
Currently what we decided in in `vitessce-data` is that null is allowed. Ilan and I have been talking about how to represent dimensions better, but I think we should allow this for the time being. https://github.com/hubmapconsortium/vitessce-data/pull/90